docs/getting_started.rst

```diff
@@ -48,6 +48,28 @@ multiple database URLs, separated by whitespace::
   # application may ask jardin to switch to other connections on the list by
   # calling 'jardin.reset_session()'.
 
+
+You can also setup database connections with connection pools (only with Postgres for now).
+See (https://www.psycopg.org/docs/pool.html)
+To do this, you need to specify the connection pools config in `CONNECTION_POOLS` like so::
+
+  # jardin_conf.py
+
+  DATABASES = {
+    'my_replica_database': 'https://username:password@replica_database.url:port'
+  }
+
+  CONNECTION_POOLS = {
```

Comment:
@mathieuripert I wonder if we should use env-var so that we can control on this on the service level. Would this be set on the app level?

Reply:
Yes we can set the pool config whenever we want and use env variables